app/src/androidTest/java/net/skyscanner/backpack/dialog/BpkDialogTest.kt

```diff
@@ -196,34 +231,19 @@ class BpkDialogTest : BpkSnapshotTest() {
       }
   }
 
-  @Test
-  fun screenshotTestFlareDialog() {
-    val bitmap = Picasso.get().load("file:///android_asset/dialog_sample.jpg").get()
-    val asyncScreenshot = prepareForAsyncTest()
-
-    val dialog = BpkDialog(activity, BpkDialog.Style.FLARE).apply {
-      title = "You are going to Tokyo!"
-      description = "Your flight is all booked."
-      icon = BpkDialog.Icon(
-        R.drawable.bpk_tick,
-        context.getColor(R.color.bpkMonteverde),
-      )
-
-      image!!.setImageBitmap(bitmap)
-
-      addActionButton(
-        BpkButton(context).apply {
-          text = "Continue"
+  private fun View.findScrollView(): ScrollView? {
```

Comment:
didn't want to expose an id for the scrollview as clients shouldn't really access it, so added a function to find the scrollview instead

Reply:
Seems like a good approach to me. We do something similar in iOS where we access private selectors by name. Not appropriate for production code, but for tests it's 💯